app/controllers/chat_channels_controller.rb

```diff
@@ -32,12 +32,12 @@ def show
 
   def create
     authorize ChatChannel
-    @chat_channel = ChatChannelCreationService.new(current_user, params[:chat_channel]).create
+    @chat_channel = ChatChannels::Creation.new(current_user, params[:chat_channel]).create
```

Comment:
could these new classes become verbs? `ChatChannels::Create` instead of `::Creation` ? Also, to keep a bit of consistency, can these new services use `.call` as their interface? See https://github.com/forem/forem/blob/master/app/services/users/delete.rb

unfortunately our code base isn't consistent in that so I understand the confusion

Reply:
Ok, I've used `perform` cause is a default name for services, but I change to `call` to follow the project.
And the name of the classes I'll change too.